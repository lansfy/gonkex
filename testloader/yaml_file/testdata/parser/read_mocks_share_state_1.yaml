- name: share state for one testscase MUST change nothing
  method: GET
  path: /some/path
  request: "requestbody"
  mocksParams:
    shareState: true
  mocks:
    someservice:
      strategy: nop
