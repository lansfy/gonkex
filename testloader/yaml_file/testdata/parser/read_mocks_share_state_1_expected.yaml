- GetName: share state for one testscase MUST change nothing
  GetMethod: GET
  Path: /some/path
  GetRequest: requestbody
  GetFileName: testdata/parser/read_mocks_share_state_1.yaml
  GetLineNumber: 1
  FirstTestInFile: true
  LastTestInFile: true
  ServiceMocks:
    someservice:
      strategy: nop

